### DISCUSSION TITLE
Actual list of Vyper compiler versions

### DISCUSSION THREAD

**Discussion by vbaranov** [vyperlang/vyper#2371] [Q&A]
Is there somewhere an actual list of Vyper compiler versions as it is in case of Solidity https://solc-bin.ethereum.org/bin/list.json ? I am working on bringing Vyper contracts verification to Blockscout.

**Comment by fubuloubu** [comment]
So the releases page, might be able to pull some JSON from the github API

Otherwise, might be interested in this:
https://github.com/vyperlang/vvm

**Reply by igorbarinov** [reply]
Thanks. We've used https://api.github.com/repos/vyperlang/vyper/releases as as a source of truth
